src/screens/Dashboard/Footer/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faFacebook, faLinkedin, faInstagram, faTwitter, faYoutube
+} from "@fortawesome/free-brands-svg-icons";
+import './style.scss';
+
+export default () => (
+  <section id="footer">
+    <div className="container">
+      <div className="top">
+        <p className="contactText">Contact</p>
+        <div className="contactInfo">
+          <p className="phone">+1-234-567-8900</p>
+          <p className="email">info@growersbrains.com</p>
+        </div>
+        <p className="title">Growers Brains</p>
+        <button className="contact">Contact Us</button>
+      </div>
+      <div className="h-line" />
+      <div className="bottom">
+        <div className="bottomLeft">
+          <ul className="footerList">
+            <li><button href="#">Help</button></li>
+            <li><button href="#">Terms</button></li>
+            <li><button href="#">Privacy Policy</button></li>
+          </ul>
+          <p className="copyright"><span dangerouslySetInnerHTML={{ "__html": "&copy;" }} />2020 Growers Brains</p>
+        </div>
+        <div className="socialIcons">
+          <ul>
+            <li><button><FontAwesomeIcon icon={faFacebook} size="lg" /></button></li>
```

Comment:
These will need to be links using the <a> tag ideally. Here's an example of all that should be included for accessibility purposes. 
```
      <li class="social-list__item">
        <a href="website link here" aria-hidden="true" class="social-list__link" target="_blank" rel="noopener" alt="go to Instagram">
          <span class="sr-only">Opens new window</span>
          <i class="fab fa-instagram"></i>
        </a>
      </li>
```

Reply:
used this as reference for social links and added cdn instead of npm package